tests/acceptance/features/webUIUpload/upload.feature

```diff
@@ -63,35 +65,35 @@ Feature: File Upload
 
   @smokeTest @ocisSmokeTest
   Scenario: uploading a big file (when chunking is implemented this upload should be chunked)
-    Given a file with the size of "30000000" bytes and the name "big-video.mp4" has been created locally
+    Given a file with the size of "30000000" bytes and the name "big-video.mp4" has been created locally in the server
     When the user uploads a created file "big-video.mp4" using the webUI
     Then no message should be displayed on the webUI
     And file "big-video.mp4" should be listed on the webUI
-    And as "Alice" the content of "big-video.mp4" should be the same as the content of local file "big-video.mp4"
+    And as "Alice" the content of "big-video.mp4" should be the same as the content of local file "big-video.mp4" in the server
```

Comment:
```suggestion
    And as "Alice" the content of "big-video.mp4" should be the same as the content of local file "big-video.mp4" in the middleware
```

I can understand "in the middleware" more easily. This step compares the "local" file that the middleware created in its local file system with the file that is stored in the server.

Maybe even say:
```
    And as "Alice" the content of "big-video.mp4" on the server should be the same as the content of local file "big-video.mp4" in the middleware
```

Reply:
issue created for this, we can do this after this PR